daemons/fenced/pacemaker-fenced.c

```diff
@@ -1115,10 +1114,10 @@ update_cib_cache_cb(const char *event, xmlNode * msg)
 {
     int rc = pcmk_ok;
     xmlNode *stonith_enabled_xml = NULL;
-    const char *stonith_enabled_s = NULL;
     static gboolean stonith_enabled_saved = TRUE;
     long timeout_ms_saved = stonith_watchdog_timeout_ms;
     gboolean need_full_refresh = FALSE;
+    bool value;
```

Comment:
Even though we only use it when rc==pcmk_rc_ok, I'm betting some static analysis will complain about it being used without being initialized, so probably worth always initializing the bools

Reply:
Fixed, and initialized all the new rc variables as well.